packages/strapi-plugin-users-permissions/config/layout.js

```diff
@@ -20,5 +20,22 @@ module.exports = {
         className: 'd-none'
       }
     }
+  },
+  admin: {
```

Comment:
Is this really needed?
This layout doesn't refer to any existing model plus I think it will mess with the schema object needed for the ctm that it already a bit buggy.